rector.php

```diff
@@ -30,5 +31,8 @@
     ]);
     $config->skip([
         PreferPHPUnitThisCallRector::class,
+        ChangeAndIfToEarlyReturnRector::class => [
```

Comment:
From my point of view you can remove this rule for every class. Can also be done in a later PR.